<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="7" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-20T17:49:09 IST" name="Suite" finished-at="2024-12-20T17:49:25 IST" duration-ms="16126">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorvalidationsTest.errorValidationTest()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" name="errorValidationTest" class="actualTests.ErrorvalidationsTest"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" name="submitOrder" class="actualTests.SubmitOrderTest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2024-12-20T17:49:09 IST" name="ErrorValidation Test" finished-at="2024-12-20T17:49:24 IST" duration-ms="14840">
      <class name="actualTests.ErrorvalidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:09 IST" name="launchApplication" finished-at="2024-12-20T17:49:13 IST" duration-ms="3740" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="errorValidationTest()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:13 IST" name="errorValidationTest" finished-at="2024-12-20T17:49:16 IST" duration-ms="3272" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at actualTests.ErrorvalidationsTest.errorValidationTest(ErrorvalidationsTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorValidationTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:16 IST" name="tearDown" finished-at="2024-12-20T17:49:16 IST" duration-ms="110" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:16 IST" name="launchApplication" finished-at="2024-12-20T17:49:18 IST" duration-ms="1627" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="errorValidationTest()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:18 IST" name="errorValidationTest" finished-at="2024-12-20T17:49:21 IST" duration-ms="3149" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at actualTests.ErrorvalidationsTest.errorValidationTest(ErrorvalidationsTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorValidationTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:21 IST" name="tearDown" finished-at="2024-12-20T17:49:21 IST" duration-ms="105" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:21 IST" name="launchApplication" finished-at="2024-12-20T17:49:24 IST" duration-ms="2520" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="fakeTest()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:24 IST" name="fakeTest" finished-at="2024-12-20T17:49:24 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fakeTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:actualTests.ErrorvalidationsTest@55de24cc]" started-at="2024-12-20T17:49:24 IST" name="tearDown" finished-at="2024-12-20T17:49:24 IST" duration-ms="112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- actualTests.ErrorvalidationsTest -->
    </test> <!-- ErrorValidation Test -->
    <test started-at="2024-12-20T17:49:09 IST" name="SubmitOrder Test" finished-at="2024-12-20T17:49:25 IST" duration-ms="16126">
      <class name="actualTests.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:09 IST" name="launchApplication" finished-at="2024-12-20T17:49:13 IST" duration-ms="3877" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:13 IST" name="submitOrder" data-provider="getData" finished-at="2024-12-20T17:49:17 IST" duration-ms="3699" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Random@me1, email=random1@gmail.com, productName=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:17 IST" name="tearDown" finished-at="2024-12-20T17:49:17 IST" duration-ms="61" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:17 IST" name="launchApplication" finished-at="2024-12-20T17:49:19 IST" duration-ms="2487" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:19 IST" name="submitOrder" data-provider="getData" finished-at="2024-12-20T17:49:23 IST" duration-ms="3764" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Random@me0, email=random@email.com, productName=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:23 IST" name="tearDown" finished-at="2024-12-20T17:49:23 IST" duration-ms="62" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:23 IST" name="launchApplication" finished-at="2024-12-20T17:49:25 IST" duration-ms="1602" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistory()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:25 IST" name="OrderHistory" finished-at="2024-12-20T17:49:25 IST" depends-on-methods="actualTests.SubmitOrderTest.submitOrder" duration-ms="403" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistory -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:actualTests.SubmitOrderTest@cc43f62]" started-at="2024-12-20T17:49:25 IST" name="tearDown" finished-at="2024-12-20T17:49:25 IST" duration-ms="68" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- actualTests.SubmitOrderTest -->
    </test> <!-- SubmitOrder Test -->
  </suite> <!-- Suite -->
</testng-results>
